shell/src/main/java/alluxio/master/MasterHealthCheckClient.java

```diff
@@ -193,11 +180,9 @@ public ProcessCheckRunnable(String alluxioMasterName) {
 
     @Override
     public void run() {
-      UserState userState = UserState.Factory.create(mConf);
-      MasterInquireClient client = MasterInquireClient.Factory.create(mConf, userState);
       try {
         while (true) {
-          List<InetSocketAddress> addresses = client.getMasterRpcAddresses();
+          List<InetSocketAddress> addresses = ConfigurationUtils.getMasterRpcAddresses(mConf);
```

Comment:
The real fix is here, the rest is IntelliJ warnings cleanup.

Reply:
Does the logic change for ZkMasterInquireClient? Since that looks like it is getting a config from zookeeper? Or is that a static list as well?